content/al/0006-produto-interno-ortogonalidade.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Produto interno e ortogonalidade
+description: >-
+  Produto interno usual (real e complexo)
+  Propriedades do produto interno
+  Conceitos associados ao produto interno
+  Vetores ortogonais
+  Conjuntos Ortogonais
+  Ortogonalização de Gram-Schmidt
+  Projeção ortogonal
+
+path: /al/produto-interno-ortogonalidade
+type: content
+---
+
+# Produto interno e ortogonalidade
+
+(baseado no material disponibilizado pela professora Esmeralda Dias)
+
+Apesar da introdução a álgebra linear ter sido realizada até agora através de sistemas lineares, existe uma diversidade enorme de aplicações geométricas desta área.
+Podemos por isso, generalizar muitas noções geométricas a qualquer espaço linear, não só a $$ \R^n$$ mas a qualquer espaço linear.
+
+## Produto interno usual em $$ \R^n$$
+
+O produto interno usual em $$ \R^n$$ pode ser definido simplesmente por
+
+:::info[Definição]
+Sejam x, y vetores de $$ \R^n$$
+
+Tal que $$ x=(x_1,x_2,x_3,... , x_n) $$
+
+e $$y=(y_1,y_2,y_3,..., y_n)$$
+
+$$ \langle x,y \rang =x_1y_1 +x_2y_2+...+x_ny_n$$
```

Comment:
```suggestion
$$
\langle x,y \rang =x_1y_1 +x_2y_2+...+x_ny_n
$$
```

To make a math block, `$$` tags must be on their own line, otherwise KaTeX considers it inline.